.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  release-management: salesforce/npm-release-management@4
```

Comment:
@mshanemc why this change?

Reply:
nice catch.  That was a branch that's been merged so it can go back to normal.